pkg/portal/graphql/app_mutation.go

```diff
@@ -521,6 +521,10 @@ var createAppInput = graphql.NewInputObject(graphql.InputObjectConfig{
 			Type:        graphql.NewNonNull(graphql.String),
 			Description: "ID of the new app.",
 		},
+		"projectWizardData": &graphql.InputObjectFieldConfig{
+			Type:        graphql.NewNonNull(ProjectWizardData),
```

Comment:
This should be nullable.

Reply:
Why? I think we must start project wizard for new projects?

Reply:
I think project wizard is a business logic of the portal frontend. The backend should not require project wizard.